### PR TITLE
feat(ontology): separate list of ontology properties (DSP-1364)

### PR DESCRIPTION
**Pull request by kilchenmann**
resolves DSP-1364

This is a first version of a separate list of properties. In another task we will add a filter and sort button.

### CODE REVIEW

File: src/app/project/ontology/ontology.component.scss [hunk 1]
```diff
@@ -45,15 +44,29 @@ $width: 340px;
...
+
+    button {
+      width: 100%;
+      text-align: left;
+    }
+  }
+
   .ontology-editor-canvas {
     min-height: calc(100vh - #{$sub-header-height} - #{$header-height} - #{$tab-bar-margin} - 100px);
+    // background: $primary_200;
```
Comment (flavens) [inline src/app/project/ontology/ontology.component.scss:58]: when there are unused css classes, we should clean them up or add a comment (remark valid for all the scss files)

**Review by flavens — APPROVED** [review]

File: src/app/project/ontology/ontology.component.scss (hunk 1 above)
Comment (kilchenmann) [reply src/app/project/ontology/ontology.component.scss:58]: Some of the css classes are not defined in the scss file, that's true. But they are needed for the tests.

Or are you talking about the commented css property here? I can clean them up...

Comment (kilchenmann) [reply src/app/project/ontology/ontology.component.scss]: Done in ac7c3cf

**Comment by mdelez** [comment]
Screen Shot 2021-02-24 at 12 23 37

The numbering seems to be a little messed up 🤔

**Comment by kilchenmann** [comment]
> Screen Shot 2021-02-24 at 12 23 37
> 
> The numbering seems to be a little messed up 🤔

Yes I know. I'm using the number from `gui order`. This Gui order is defined in the ontology; it's part of the res class cardinality. In case of missing or doubled numbers it's a mistake in the ontology. The test data are not well done.

**Review by mdelez — APPROVED** [review]
I would stress that we need to have some unit tests for the ontology component but I don't think this is the PR for that so I'll approve this one :)

**Comment by kilchenmann** [comment]
> I would stress that we need to have some unit tests for the ontology component but I don't think this is the PR for that so I'll approve this one :)

I'm working on it 😉

**Pull request merged**